src/Loadbalancers/AclClient.php

```diff
@@ -13,6 +19,32 @@ class AclClient extends BaseClient
         'backend_id' => 'backendId',
     ];
 
+    const MATCH_MAP = [
+        'header_id' => 'headerId',
+        'match_case' => 'matchCase',
+        'path_id' => 'pathId',
+    ];
+
+    const HEADER_MAP = [];
+
+    const FREETYPE_MAP = [];
+
+    const CONDITION_MAP = [
+        'backend_id' => 'backendId',
+    ];
+
+    const ACTION_MAP = [
+        'arg1' => 'firstArg',
+        'arg2' => 'secondArg',
+        'arg3' => 'thirdArg',
+    ];
+
+    const IP_MAP = [
+        'ip_address' => 'address'
```

Comment:
```suggestion
        'ip_address' => 'address',
        'acl_id' => 'aclId'
```

This needs adding to the map